Java/core/src/main/java/com/amazon/randomcutforest/tree/AbstractCompactRandomCutTree.java

```diff
@@ -52,6 +54,8 @@
      */
     public static final int NULL = -1;
 
+    public static final int switchNodeStore = 16383;
```

Comment:
I think this would be more self-explanatory if it was a constant in the SmallNodeStore class: `SmallNodeStore.MAX_TREE_SIZE` or something like that.

Reply:
Done